libs/server/core/application-services/src/lib/notification-digest/notification-digest.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from "@nestjs/common";
+import { ActivityPostRepository, CommentRepository, LikeRepository } from "@involvemint/server/core/domain-services";
+import { ActivityPost, ActivityPostQuery, CreateActivityPostDto, DisableActivityPostDto, EnableActivityPostDto, LikeActivityPostDto, likeQuery, UnlikeActivityPostDto } from "@involvemint/shared/domain";
+import { IQuery } from "@orcha/common";
+import { AuthService } from '../auth/auth.service';
+import * as uuid from 'uuid';
+import { MoreThan } from 'typeorm';
+
+@Injectable()
+export class NotificationDigestService {
+    constructor(
+        private readonly auth: AuthService,
+        private readonly activityPostRepo: ActivityPostRepository,
+        private readonly likeRepo: LikeRepository,
+        private readonly commentRepo: CommentRepository
+    ) {}
+
+    // async getRecentLikes(recentPosts: IParserObject<ActivityPost, IQueryObject<ActivityPost>>[], lastDays:Date) {
+    //   const recentLikes: any = [];
+    //   recentPosts.forEach( async (post) => {
+    //     let likes = await this.likeRepo.query({id: true, user: {id: true}}, 
+    //                                           {where: { dateCreated: MoreThan(lastDays), activityPost: post.id}})
```

Comment:
We should just delete this file completely? Unless we expect to do more complex notifications later on?

Reply:
if this was a relic from a previous attempt at notifications (i think it was), then yes, let's definitely remove this